supervisor/addons/__init__.py

```diff
@@ -169,7 +169,7 @@ async def install(self, slug: str) -> None:
             await addon.instance.install(store.version, store.image)
         except DockerError as err:
             self.data.uninstall(addon)
-            raise AddonsError() from err
+            raise AddonsError(err) from err
```

Comment:
Do we need raise from err if we own the error? Important that our exception chain checker work with the new structure as well

Reply:
Not sure, it should, I'll add a test for it

Reply:
Actually, check the current tests for `check_exception_chain` it seems to cover all cases already?